src/pages/index.tsx

```diff
@@ -1,118 +1,56 @@
-import Image from 'next/image'
-import { Inter } from 'next/font/google'
+import Image from "next/image";
+import { Inter } from "next/font/google";
```

Comment:
❌ 
you should put eslint/prettier changes in its own commit, or better in its own PR

Reply:
I have no idea why those changed, I didn't do that

Reply:
then you should not commit them,
if you want to do Es-lint fixes do them in a seperated commit